include/revng/Pipeline/Loader.h

```diff
@@ -0,0 +1,202 @@
+#pragma once
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <any>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/StringMap.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/YAMLTraits.h"
+
+#include "revng/Pipeline/LLVMContainer.h"
+#include "revng/Pipeline/LLVMPipe.h"
+#include "revng/Pipeline/PureLLVMPipe.h"
```

Comment:
not really, it's part of the grammar for simplicity of use. To avoid this we would need much stronger configuration that requires extensible grammars.

Reply:
Not today.